.travis.yml

```diff
@@ -60,23 +57,32 @@ stages:
 
 jobs:
   include:
-    - stage: tests
-      name: "Backend tests"
-      script: ./ciBackendTest.sh $TRAVIS_SCALA_VERSION
-      after_success: if [ "$COVERAGE" = true ]; then sbt ++$TRAVIS_SCALA_VERSION coveralls; else echo "Skipping coveralls"; fi
+    - stage: test
+      name: "Backend tests 2.11"
+      script: ./ciBackendTest.sh 2.11.12
+      after_success: if [ "$COVERAGE" = true ]; then sbt ++2.11.12 coveralls; else echo "Skipping coveralls"; fi
+    - name: "UI slow tests 2.11"
+      script: sbt ++2.11.12 clean ui/slow:test
```

Comment:
Why not to just one crossbuild task with single plus?

Reply:
Well, in theory it gives us better parallelism... I can try + version